manifests/server/config.pp

```diff
@@ -36,21 +36,26 @@
   }
 
   #Debian: Creating world readable directories before installing.
-  if $managed_dirs {
-    $managed_dirs.each | $entry | {
-      $dir = $options['mysqld']["${entry}"]
-      if ( $dir and $dir != '/usr' and $dir != '/tmp' ) {
-        exec {"${entry}-managed_dir-mkdir":
-          command => "/bin/mkdir -p ${dir}",
-          unless  => "/usr/bin/dpkg -s ${mysql::server::package_name}",
-          notify  =>  Exec["${entry}-managed_dir-chmod"],
-        }
-        exec {"${entry}-managed_dir-chmod":
-          command     => "/bin/chmod 777 ${dir}",
-          refreshonly => true,
+  case $::operatingsystem {
```

Comment:
Thank you @evgenkisel for incorporating the comments. By adding the operating system specific it will fail on other oses. So we may have to include conditions for other OSes too to avoid regression. Please let us know if you need more information. Thanks for your contribution.

Reply:
Got it. This code is supposed to run only on Debian. Perfect. Running the release checks across all OS and verifying the changes. Thanks again.